crates/bin/pd/src/migrate/testnet78.rs

```diff
@@ -161,10 +182,12 @@ async fn truncate_validator_fields(delta: &mut StateDelta<Snapshot>) -> anyhow::
 ///   * Governance Signaling Proposals:
 ///    - `commit hash` (255 bytes)
 async fn truncate_proposal_fields(delta: &mut StateDelta<Snapshot>) -> anyhow::Result<()> {
+    tracing::info!("truncating proposal fields");
     let next_proposal_id: u64 = delta.next_proposal_id().await?;
 
     // Range each proposal and truncate the fields.
     for proposal_id in 0..next_proposal_id {
+        tracing::info!("truncating proposal: {}", proposal_id);
```

Comment:
@zbuc doesn't this have the same issue of pulling data using stronger rules that existed when it was stored?

Reply:
Yes, will follow up in a new PR

Reply:
https://github.com/penumbra-zone/penumbra/pull/4606